nav2_mppi_controller/include/nav2_mppi_controller/optimizer.hpp

```diff
@@ -202,8 +196,8 @@ class Optimizer
    * @param state fill state
    */
   void integrateStateVelocities(
-    xt::xtensor<float, 2> & trajectories,
-    const xt::xtensor<float, 2> & state) const;
+    Eigen::Array<float, -1, 3> & trajectories,
```

Comment:
Is there a typedef for -1 and dynamic so that an onlooker knows this means dynamic size? I.e. `Eigen::Dynamic`

Reply:
We can use Eigen::Dynamic instead of -1, that would be more self-explanatory.